packages/router/README.md

```diff
@@ -143,6 +128,41 @@ becomes...
 </MainLayout>
 ```
 
+### `private` Set
+
+All `Routes` nested in `<Private>` require authentication.
```

Comment:
This should probably come later; a bit of misdirection to title the section "`private` Set", have the first sentence (usually the topic, etc.) be about the `<Private>` component, then talk about Sets in the next paragraph. I'll make a more actionable edit!